src/utils/http-client/get-modules/http.ts

```diff
@@ -76,6 +76,7 @@ function parseUrlAndOptions(urlStr: string, baseOptions: RequestOptions) {
     protocol: url.protocol,
     hostname,
     path: `${url.pathname || ""}${url.search || ""}`,
+    agent: false,
```

Comment:
I think options should be able to be overridden.

```js
  const options: RequestOptions = {
    agent: false,
    ...baseOptions,
    protocol: url.protocol,
    hostname,
    path: `${url.pathname || ""}${url.search || ""}`,
```

Reply:
Ah, of course! Thank you.